### PR TITLE
Setting up GSoD 2023

### PR DESCRIPTION
**Pull request by nate-double-u**

### CODE REVIEW

**Comment by nate-double-u** [comment]
[GSoC Admin Guide](https://developers.google.com/season-of-docs/docs/admin-guide)

**Comment by nate-double-u** [comment]
I'm asking the projects to have their proposals in by March 14, ten days before Google requires them so that we have time to edit and deliberate.

**Review by jaytiaki — APPROVED** [review]
/lgtm All straightforward.

File: seasonofdocs/README.md [hunk 1]
```diff
@@ -1,98 +1,28 @@
...
-Date | Activity
----|---
-March 26, 2021 | The CNCF has submitted a [project proposal](#project-proposal) for consideration.
+# Season of Docs
 
-All steps are based on the [GSoD 2021 timeline](https://developers.google.com/season-of-docs/docs/timeline).
+Google [Season of Docs](https://developers.google.com/season-of-docs) gives technical writers an opportunity to work with open source projects.
 
-## Project Proposal
+The CNCF is applying to participate in the Season of Docs as an org in 2023. We'll announce more details pending our acceptance into the program.
```
Comment (FeynmanZhou) [inline seasonofdocs/README.md:5]: @nate-double-u  Hi Nate, when will GSoD 2023 be opened for CNCF maintainers to submit the project proposals?

Comment (nate-double-u) [reply seasonofdocs/README.md:5]: Sorry i was unclear, you can submit proposals now. Here's the announcement: https://github.com/cncf/mentoring/discussions/843

**Pull request merged**